internal/cmd/operator-sdk/alpha/config3alphato3/convert_config_3-alpha_to_3_test.go

```diff
@@ -0,0 +1,130 @@
+			Expect(string(output)).To(Equal(expectedCfgStr))
+		},
+		Entry("basic", basicConfig, basicConfigExp),
+		Entry("no resources", noResourcesConfig, noResourcesConfigExp),
+		Entry("multiple resources", multipleAPIsConfig, multipleAPIsConfigExp),
+	)
+})
+
+const (
+	basicConfig = `domain: example.com
+layout: ansible.sdk.operatorframework.io/v1
+projectName: memcached-operator
+resources:
+- crdVersion: v1
+  group: cache
+  kind: Memcached
+  version: v1alpha1
+version: 3-alpha
+`
+	basicConfigExp = `domain: example.com
+layout: ansible.sdk.operatorframework.io/v1
+projectName: memcached-operator
+resources:
+- api:
+    crdVersion: v1
+    # TODO(user): Uncomment the below line if this resource's CRD is namespace scoped, else delete it.
+    # namespaced: true
+  # TODO(user): Uncomment the below line if this resource implements a controller, else delete it.
+  # controller: true
+  domain: example.com
+  group: cache
+  kind: Memcached
+  version: v1alpha1
+version: "3"
+`
+)
+
+const (
+	noResourcesConfig = `domain: example.com
+layout: ansible.sdk.operatorframework.io/v1
+projectName: memcached-operator
+version: 3-alpha
+`
+	noResourcesConfigExp = `domain: example.com
+layout: ansible.sdk.operatorframework.io/v1
+projectName: memcached-operator
+version: "3"
+`
+)
+
+const (
+	multipleAPIsConfig = `domain: example.com
+layout: ansible.sdk.operatorframework.io/v1
+projectName: memcached-operator
+resources:
+- crdVersion: v1
+  group: cache
+  kind: Memcached
+  version: v1alpha1
+  webhookVersion: v1
+- crdVersion: v1
+  group: cache
+  kind: MemcachedRS
+  version: v1alpha1
+version: 3-alpha
+`
+	multipleAPIsConfigExp = `domain: example.com
+layout: ansible.sdk.operatorframework.io/v1
+projectName: memcached-operator
+resources:
+- api:
+    crdVersion: v1
+    # TODO(user): Uncomment the below line if this resource's CRD is namespace scoped, else delete it.
+    # namespaced: true
+  # TODO(user): Uncomment the below line if this resource implements a controller, else delete it.
+  # controller: true
+  domain: example.com
+  group: cache
+  kind: Memcached
+  version: v1alpha1
+  webhooks:
+    # TODO(user): Uncomment the below line if this resource's webhook implements a conversion webhook, else delete it.
+    # conversion: true
+    # TODO(user): Uncomment the below line if this resource's webhook implements a defaulting webhook, else delete it.
+    # defaulting: true
+    # TODO(user): Uncomment the below line if this resource's webhook implements a validating webhook, else delete it.
+    # validation: true
+    webhookVersion: v1
+- api:
+    crdVersion: v1
+    # TODO(user): Uncomment the below line if this resource's CRD is namespace scoped, else delete it.
+    # namespaced: true
+  # TODO(user): Uncomment the below line if this resource implements a controller, else delete it.
+  # controller: true
+  domain: example.com
+  group: cache
+  kind: MemcachedRS
+  version: v1alpha1
+version: "3"
```

Comment:
/hold 

It is not covering all data/scenarios.

Reply:
What scenarios does it not cover?

Reply:
I added above. You answered. I have only one concern.